### PR TITLE
wip: imporve error message of js spl-token client

### PR DESCRIPTION
**Pull request by ryoqun**
sadly, this isn't helpful.

```
Error: Failed to find account
     at Token.getAccountInfo (node_modules/@solana/spl-token/lib/index.cjs.js:474:13)
     at runMicrotasks (<anonymous>)
     at processTicksAndRejections (internal/process/task_queues.js:97:5)
     at async Token.getOrCreateAssociatedAccountInfo node_modules/@solana/spl-token/lib/index.cjs.js:319:16)
```

### CODE REVIEW

File: token/js/client/token.js [hunk 1]
```diff
@@ -764,13 +768,13 @@ export class Token {
   ): Promise<AccountInfo> {
     const info = await this.connection.getAccountInfo(account, commitment);
     if (info === null) {
-      throw new Error(FAILED_TO_FIND_ACCOUNT);
+      throw new Error(formatDynamicErrorMessage(FAILED_TO_FIND_ACCOUNT, account.toString());
```
Comment (ryoqun) [inline token/js/client/token.js:771]: @jstarry these days, what's most fashionable way of structured error in js land? Still, these adhoc way are rampant?

Comment (ryoqun) [reply token/js/client/token.js:771]: for two things, I want to add some `error.code = FAILED_...` and `error.context = {address}`.. For internal use.

**Comment by ryoqun** [comment]
Also, I think this is pretty low-priority. I think just library consumers can print whatever arguments they gave to these functions.

File: token/js/client/token.js (hunk 1 above)
Comment (jstarry) [reply token/js/client/token.js:771]: You can use the following syntax with backticks: `` `Failed to find account with address ${account.toString()}` ``

Comment (jstarry) [reply token/js/client/token.js:771]: I will have to do some research, this is something that could be improved in the web3 library as well!

**Pull request closed without merging**